docs/source/_templates-stable/layout.html

```diff
@@ -30,5 +30,18 @@
   gtag('config', 'UA-117752657-2');
 </script>
 
+<script>
+  function redirectAnchor() {
+    if (location.hash) { // e.g., #torch.flip
```

Comment:
If all anchors were `torch.*`, would it be safer to only redirect those anchors starting with `torch.*`? Since some other anchors may be used in future that do not need redirects and indeed point into some sections of the new pages

Reply:
Good idea! I'll add this check.

Reply:
And also in theory this redirect is needed only in handful of pages, such as `torch.html` (which may be non-existent now, but one solution would be to bring it back with only redirect in it), and maybe a few others (inlcuding templates-master though), but not at all pages, rigth?

Reply:
There is an issue https://github.com/pytorch/pytorch/issues/38010 planed to split more pages into subpages. I guess it suffices to check the name of the anchor. Also, an anchor will only trigger redirection if it points to the current page but the target is not found, so it won't hurt other pages that haven't split yet.

Reply:
One way that this could hurt is that if someone makes a typo in the anchor link, or the anchor link gets removed, then the redirect goes to a dead page: https://5578841-65600975-gh.circle-artifacts.com/0/docs/torch.html#torch.flipfoobar